client/header/activity-panel/unread-indicators.js

```diff
@@ -31,27 +30,26 @@ export function getUnreadNotes( select ) {
 	}
 	const notesQuery = {
 		page: 1,
-		per_page: 1,
+		per_page: QUERY_DEFAULTS.pageSize,
```

Comment:
Related to my above comment- it would be much better to query a single item with the items that we need instead of querying a page of them and filtering client-side.

Reply:
A TODO comment above this object would be helpful to remind us we'll need to change this once pagination is introduced.